site/content/contribute/getting-started/inactive-contribution.md

```diff
@@ -8,17 +8,17 @@ subsection: Getting Started
 This process describes how inactive contributions are managed at Mattermost, inspired by the [Kubernetes project](https://github.com/kubernetes/kubernetes):
 
 1. After 14 days of inactivity, a contribution becomes stale.
- - If action is required from submitter, Community Coordinator asks if the team can help clarify previous feedback or provide guidance on next steps, and adds `stale` label to the contribution.
- - If action is required from reviewers, Community Coordinator asks reviewers to share feedback or help answer questions, and adds `stale` label to the contribution. The Coordinator will follow up with reviewers until a response is received.
+ - If action is required from submitter, Community Coordinator asks if the team can help clarify previous feedback or provide guidance on next steps, and adds `lifecycle/0:stale` label to the contribution.
```

Comment:
0/5, but I originally proposed starting at `0` as a cute way of counting negatively. If we're sticking with positive numbers, perhaps start at `1` for uniformity with our other labels?

Reply:
+1